frontend/catalyst/qjit_device.py

```diff
@@ -294,7 +295,9 @@ def preprocess(
         execution_config: qml.devices.ExecutionConfig = qml.devices.DefaultExecutionConfig,
     ):
         """Device preprocessing function."""
-        program, config = self.original_device.preprocess(execution_config)
+        # TODO: readd the user preprocessing program once transforms are compatible with TOML files
```

Comment:
```suggestion
        # TODO: read the user preprocessing program once transforms are compatible with TOML files
```

Reply:
That's what I meant https://en.wiktionary.org/wiki/readd